docs/proposals/mvp-dashboard-design.md

```diff
@@ -43,18 +43,125 @@ can query the relevant test builds.
+test data. This json is then used to create the graph of the specified metrics
+as they trend over past builds.
 
 ## Rationale
 
-A discussion of alternate approaches and the trade offs, advantages, and
-disadvantages of the specified approach.
+### Choice to build our own front-end
+
+1. **Avoid Grafana learning curve**
+
+   [Grafana][4] is used to facilitate performance monitoring in the performance
+   test lab. The team is familiar with it and we have templated various
+   visualizations for tracking the relevant metrics from each node in our
+   cloud. Using Grafana still requires time to use, time that other teams do
+   not have to invest. Our goal with the perf-dash is to allow other teams to
+   access the page and quickly be able make queries and comparisons between
+   different test runs. By minimizing the learning curve, we raise the
+   probability that other teams will want to use our product.
+
+1. **Need for metrics comparison of test runs across multiple labs**
+
+   Grafana lacks the ability to directly compare multiple metrics from
+   different test runs from different times. Since one of our goals is to create
+   a tool that can be used to compare test runs across different lab
+   environments it became clear that Grafana would not be able to fulfill our
+   demands.
+
+#### Alternatives
+
+- *Any proposed alternatives and their arguments should be located here.*
+
+### Choice of data storage
+
+1. **Need for SQL-like querying with optimizations**
+
+   [InfluxDB][1] is a database service optimized for temporal data, which is a
+   characteristic common to performance test data. The querying language
+   used is similar to SQL but incorporates time stamps and indexes off of
+   these whenever filtering. This type of optimization is a benefit to
+   the performance monitoring in this proposal because of the nature of
+   performance analysis.
+
+   Performance analysis requires the investigation of a series of metrics in a
+   test as they trend across time, this results in the high likelihood of
+   queried data being located close on the disk. These same time-based
+   reasoning applies for writing to disk, a write of performance metrics is
+   very likely followed by a similar metrics at a time interval ahead in time;
+   the implication being that the next write will occur close by on the disk.
+
+1. **Established database with historic performance data**
+
+   Previous performance data taken by the team is currently stored in an
+   InfluxDB server. Uniformity between our past, current, and future datasets
+   will ensure that we can make comparisons between different test results.
+   We can accomodate new and different data by properly using tags, a feature
+   of the InfluxDB query language.
 
 ## Implementation
 
-A description of the steps in the implementation, who will do them, and when.
+@meteorfox has volunteered to set up the scaffolding for both the server and
```

Comment:
I dont think we should have names, especially since I may want to help work on the scaffolding as well (for learning purposes)